keyboards/crkbd/keymaps/ericgebhart/rules.mk

```diff
@@ -0,0 +1,2 @@
+OLED_ENABLE = yes
+OLED_DRIVER = SSD1306
```

Comment:
This is the default, actually.

Reply:
Sometimes I turn it off. I guess it doesn't really matter.

Reply:
removed it.